Assignments/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
+services:
+  assignment5:
+    container_name: assignment5
+    depends_on:
+      - mariadb
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./docker_hw
+
+  mariadb:
+    container_name: mariadb
+    image: mariadb:10.6.4
+    restart: always
+    environment:
+      MYSQL_USER: root
+      MYSQL_PASSWORD: BDAMaster
+      MYSQL_ROOT_PASSWORD: BDAMaster
+    ports:
+      - "8080:8080"
```

Comment:
I ran into a bunch of trouble using 3306 since I think it interfered with the server running on my local machine? Maybe that's causing some issue for you? Also, the quotes around the ports here may not be necessary. I don't have those.